app/bundles/CampaignBundle/Translations/en_US/messages.ini

```diff
@@ -225,3 +225,14 @@ mautic.config.peak_interaction_timer.fetch_interactions_from.tooltip='Select the
 mautic.config.peak_interaction_timer.fetch_limit.tooltip='Set the maximum number of interactions of each type (e.g., page hits, email reads) to retrieve for timing optimization.'
 mautic.campaign.parent.details='The event followed the <strong>%path%</strong> path of the %type% "<strong>%name%</strong>". Here are the details:'
 mautic.campaign.event.path.tooltip='This event followed the %path% path.'
+mautic.campaign.timeline.user_name="User Name"
+mautic.campaign.timeline.event_type="Action"
+mautic.campaign.timeline.event_timestamp="Timestamp"
+mautic.campaign.event.create="Campaign created"
+mautic.campaign.event.update="Campaign updated"
+mautic.campaign.timeline.toggle_details="Toggle details."
```

Comment:
Let's re-use `mautic.lead.timeline.toggle_details`. I know I'm breaking the rule of encapsulation but the campaign bundle cannot work without the lead bundle anyway.